components/molecules/charts/highCharts/index.js

```diff
@@ -1,18 +1,61 @@
 /**
  * Molecules - Charts - Highcharts
  */
+import { any, arrayOf, bool, func, oneOf } from 'prop-types'
 
 // Highcharts
 import HighStockcharts from 'highcharts/highstock'
 import Highcharts from 'highcharts'
+import HighMapcharts from 'highcharts/highmaps'
+import HighGanttcharts from 'highcharts/highcharts-gantt'
 import HighchartsReact from 'highcharts-react-official'
 
-export const HighChart = ({ options, constructorType, chartType }) => {
+export const HighChart = ({
+  allowChartUpdate,
+  callback,
+  constructorType,
+  containerProps,
+  immutable,
+  options,
+  updateArgs,
+  ...props
+}) => {
+  let chart
+  switch (constructorType) {
+    case 'stockChart':
+      chart = HighStockcharts
+      break
+    case 'mapChart':
+      chart = HighMapcharts
+      break
+    case 'ganttChart':
+      chart = HighGanttcharts
+      break
+    default:
+      chart = Highcharts
+      break
+  }
```

Comment:
Please create a function outside the component function like `getChartType()` or something similar, and do the decision there.